### PR TITLE
Docs demo 1

### PR DESCRIPTION
**Pull request by corinnebosley**

### CODE REVIEW

File: doc/developingCubeBrowsers/interactions_2.ipynb [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+    "name": "ipython",
+    "version": 2
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython2",
+   "version": "2.7.11"
+  },
```
Comment (corinnebosley) [inline doc/developingCubeBrowsers/interactions_2.ipynb]: I removed all outputs and closed all widgets before saving and pushing.  I don't know how I can make this any smaller now.

**Comment by marqh** [comment]
reopens #50, which was closed in error

File: doc/developingCubeBrowsers/interactions_2.ipynb [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# This filepath is a directory containing the set of options for the first selection box.\n",
+    "file_options = glob.glob(os.path.join(iris.config.SAMPLE_DATA_DIR, '*')) \n",
+    "\n",
```
Comment (corinnebosley) [inline doc/developingCubeBrowsers/interactions_2.ipynb]: @marqh This (above) is what I used instead of iris.sample_data_path() on Bill's advice.

**Comment by marqh** [comment]
excellent, thanks @corinnebosley

**Pull request merged**